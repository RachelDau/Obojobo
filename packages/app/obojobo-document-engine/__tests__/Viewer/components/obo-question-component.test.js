import React from 'react'
import renderer from 'react-test-renderer'
import OboQuestionComponent from '../../../src/scripts/viewer/components/obo-question-component'

describe('OboQuestionComponent', () => {
	test('Renders as expected', () => {
		const componentContents = <OboQuestionComponent />
		const component = renderer.create(componentContents)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})

	test('Static method returns null', () => {
		expect(OboQuestionComponent.getQuestionAssessmentModel()).toBe(null)
	})
})
