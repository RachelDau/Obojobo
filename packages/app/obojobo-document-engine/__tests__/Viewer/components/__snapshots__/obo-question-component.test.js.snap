// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OboQuestionComponent Renders as expected 1`] = `null`;
